packages/plugins/ENS/src/SNSAdaptor/SearchResultInspector.tsx

```diff
@@ -126,6 +126,15 @@ export const SearchResultInspectorContent = forwardRef(function (
     })
 
     const badgeCollectionRef = useRef<HTMLDivElement>(null)
+
+    useEffect(() => {
+        setIsShowSocialAccountList(
```

Comment:
The mutation of `ref.current` is undetected sometimes if use it directly. It's okay to set state in `useEffect` if it is not a cycling.